pkgs/development/libraries/v8/3.16.14.nix

```diff
@@ -58,7 +59,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     install -vD out/Release/d8 "$out/bin/d8"
     ${if stdenv.system == "x86_64-darwin" then ''
```

Comment:
you can use `stdenv.isDarwin` instead of looking at the system attribute directly.